installing_invest_python_package.md

```diff
@@ -0,0 +1,56 @@
+# Installing the InVEST python package
+
+## Suggested method
+
+**Pattern:**
+```
+conda create -y -n <name> python=<python version>
+conda activate <name>
+conda install -y gdal=<gdal version>
+pip install natcap.invest[ui]==<invest version>
+```
+Replace `<name>` with any name you'd like to give your environment.
+Replace `<python version>` with a python version known to be compatible with the desired invest version.
+Replace `<gdal version>` with a GDAL version known to be compatible with the desired invest version.
+Replace `<invest version>` with the desired invest version.
+
+Most of the time, it is not really necessary to specify the versions of `python`, `gdal`, and `natcap.invest`. If you do not specify a version, the latest version will be installed. Usually the latest versions are compatible with each other, but not always. Specifying versions that are known to work can prevent some problems.
+
+This is not the only way to do it. There are other virtual environment tools and other ways to install GDAL. 
+
+**Example for InVEST 3.9.1:**
+```
+conda create -y -n invest391 python=3.9.7
+conda activate invest391
+conda install -y gdal=3.3.1
+pip install natcap.invest[ui]==3.9.1
+```
+
+**Condensed into one line:**
+```
+conda create -y -n invest391 python=3.9.7 gdal=3.3.1 && conda activate invest391 && pip install natcap.invest[ui]==3.9.1
+```
+
+## Details
+Here is an explanation of what the commands are doing:
+### 1. Create a brand-new environment with the correct python version.
+   `conda create -y -n <name> python=<python version>`
+   
+   To be safe, you should **always install `invest` into a brand-new virtual environment**. This way you can be sure you have all the right versions of dependencies. Many issues with installing or using the `invest` package arise from dependency problems, and it's a lot easier to create a new environment than it is to fix an existing one.
+   
+### 2. Activate the brand-new environment just created.
+   `conda activate <name>`
+
+   If you run `conda list` after this, you'll see the specified python version is there along with around 15 other packages that are included with python by default. None of these are specific to invest. You're now in an isolated environment so you can control which versions of dependencies are available to invest.
+   
+### 3. Install GDAL before installing invest
+   `conda install -y gdal=<gdal version>`
```

Comment:
I haven't tried with the defaults channel lately. Is it important to point to `conda-forge`?

Reply:
Good point